distribution/docs/src/main/resources/content/_running/starting-intro.adoc

```diff
@@ -226,3 +226,65 @@ If the system fails to start automatically on boot with any messages about the J
 
 Because ${branding} is built on top of Apache Karaf, more information on operating ${branding} can be found in the http://karaf.apache.org/index/documentation.html[Karaf documentation].
 
+
+==== Automatically Starting Solr as a Process
+
+If ${branding} is configured to start as a service automatically on system boot, then Solr will also need to be configured to start automatically.
+
+*Linux*
+
+From the https://lucene.apache.org/solr/guide/7_2/taking-solr-to-production.html[Solr documentation]:
```

Comment:
Did I miss a version reference here?

Reply:
https://lucene.apache.org/solr/guide/6_6/index.html

Reply:
I'm pretty sure just changing `7_2` to `6_6` is valid for that particular page.